sycl/source/detail/device_info.hpp

```diff
@@ -746,6 +762,8 @@ struct get_device_info_impl<
            matrix_type::fp32, matrix_type::fp32},
           {8, 0, 0, 0, 16, 16, matrix_type::bf16, matrix_type::bf16,
            matrix_type::fp32, matrix_type::fp32},
+          {8, 0, 0, 0, 16, 8, matrix_type::tf32, matrix_type::tf32,
```

Comment:
I'm not very familiar with this part of the code, so maybe this is a silly question, but if the PR is meant to add GNR matrix combinations, why do we need to update PVC matrix combinations?

Reply:
Updated description